src/ReverseProxy/Utilities/ConfigurationReadingExtensions.cs

```
@@ -23,6 +23,16 @@ internal static class ConfigurationReadingExtensions
             return configuration[name] is string value ? TimeSpan.Parse(value, CultureInfo.InvariantCulture) : null;
         }
 
+        internal static Uri ReadUri(this IConfiguration configuration, string name)
+        {
+            if (configuration[name] is string value && Uri.TryCreate(value, UriKind.Absolute, out var parsedUri))
```

We've intentionally avoided the Try APIs in the config reader because we want to actively report errors. Use the normal constructor and let it throw if the value is invalid.

---

This makes sense. I changed this in newest commit.